### ISSUE TITLE
According to your line msg, this project does not work?

### ISSUE THREAD

**Issue by tomchin** [hanc1027/vue-cordova#1]
Just to confirm our current status is:
1. (framework: Vue + Cordova, plugin: Cordova) doesn't work
2. You are troubleshooting (framework: Vue + Capacitor, plugin: Cordova)
3. (framework: Vue + Capacitor, plugin: Capacitor) works but there are less Capacitor plugins and missing source code (e.g. Motion)

The above summary is correct?

**Comment by hanc1027** [comment]
Yes! These is current status.

**Comment by tomchin** [comment]
Nelson, the project you shared with me (test.zip) is #1

Do you have a project which is #2?   Also, can you follow this section "Important: Configuration" on https://capacitor.ionicframework.com/docs/cordova/using-cordova-plugins/  to make sure to setup prorperly on IOS Info.plist https://capacitor.ionicframework.com/docs/ios/configuration, and for Android https://capacitor.ionicframework.com/docs/android/configuration?

Thanks!

**Comment by tomchin** [comment]
Did you follow https://cordova.apache.org/docs/en/latest/guide/cli/index.html to register the plugins:
- cordova platform add android ios
- cordova plugin add <plugin>
....

I think the above is needed for #2 as well

**Comment by hanc1027** [comment]
Yes! I have also followed the tips in this doc: https://capacitor.ionicframework.com/docs/cordova/using-cordova-plugins/

This is my project(Framework: Vue+ Capacitor  Plugin: Cordova): https://github.com/hanc1027/cap_cor_vue 

And both of these doc(https://capacitor.ionicframework.com/docs/ios/configuration https://capacitor.ionicframework.com/docs/android/configuration), I have seen that before.
It talks few , if we modify the value in Info.plist and AndroidManifest.xml, it will also resume the old value when we run 'npx cap add ios' or 'npx cap add android'.

---

I didn't follow the setup by this web(https://cordova.apache.org/docs/en/latest/guide/cli/index.html)
Because we need to use the vue framework.
But in the 'test project' I have used the same method to run cordova.

In the #2 project(cap_cor_vue) (framework: Vue + Capacitor, plugin: Cordova)
If we also follow the test zip's method, there will be a problem.
I compile the vue to native by 'npx cap add ios' before we follow the test zip's method.
After, there will be two way to compile the vue to native. Like as 'npx cordova run'.
Cordova is older than Capacitor.
